todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/util/providers/BaseNavigationProvider.java

```diff
@@ -0,0 +1,36 @@
+package com.example.android.architecture.blueprints.todoapp.util.providers;
+
+/**
+ * Handles navigation between Activities in the app.
+ */
+public interface BaseNavigationProvider {
```

Comment:
Moving the navigation responsibility from VM to Navigators for every feature. Keeping this trigger the actual navigation.